plugins/speculation-rules/readme.txt

```diff
@@ -23,7 +23,7 @@ A filter can be used to exclude certain URL paths from being eligible for prefet
 
 = Browser support =
 
-The Speculation Rules API is a new web API, and the specific syntax used by the plugin currently requires using Chrome 121+.
+The Speculation Rules API is a new web API, and the specific syntax used by the plugin currently requires using Chrome or Edge (specifically Chromium 121+); other Chromium-based browsers like Opera should also work, although Arc and Brave may not work if uBlock Origin is enabled.
```

Comment:
@tunetheweb I've updated the browser support section here as we've chatted about today.

Reply:
How about something simpler like "...currently requires a recent Chromium based browser like Edge or Chrome"

Reply:
Confirmed Opera does work by default.
Confirmed Arc and Brave disable this by default but not sure if we need to specifically mention ALL browsers.

WDYT about this?:

```suggestion
The Speculation Rules API is a new web API, and the functionality used by the plugin is supported in Chromium-based browsers such as Chrome, Edge, or Opera based on version 121 or above. Other browsers such as Safari and Firefox will ignore the functionality with no ill effects but will not benefit from the speculative loading. Note that extensions (for example uBlock Origin) may disable preloading by default.
```